### PR TITLE
Modernized some parts of the code base.

### PR DESCRIPTION
**Pull request by obfischer**

### CODE REVIEW

File: src/main/java/org/codehaus/mojo/versions/UseLatestReleasesMojo.java [hunk 1]
```diff
@@ -133,53 +133,41 @@ private void useLatestReleases( ModifiedPomXMLEventReader pom, Collection<Depend
...
+                            if (PomHelper.setProjectParentVersion(pom, newVersion.toString())) {
+                                getLog().debug("Made parent update from " + version + " to " + newVersion.toString());
                             }
                         }
                     }
-                    if ( PomHelper.setDependencyVersion( pom, dep.getGroupId(), dep.getArtifactId(), version,
-                                                         newVersion, getProject().getModel() ) )
-                    {
-                        getLog().info( "Updated " + toString( dep ) + " to version " + newVersion );
+                    if (PomHelper.setDependencyVersion(pom, dep.getGroupId(), dep.getArtifactId(), version,
```
Comment (olamy) [inline src/main/java/org/codehaus/mojo/versions/UseLatestReleasesMojo.java]: why changing code format?

File: src/main/java/org/codehaus/mojo/versions/UseLatestReleasesMojo.java [hunk 2]
```diff
@@ -133,53 +133,41 @@ private void useLatestReleases( ModifiedPomXMLEventReader pom, Collection<Depend
...
+            Matcher versionMatcher = matchSnapshotRegex.matcher(version);
+            if (!versionMatcher.matches()) {
+                Artifact artifact = this.toArtifact(dep);
+                if (!isIncluded(artifact)) {
                     continue;
                 }
 
-                ArtifactVersion selectedVersion = new DefaultArtifactVersion( version );
-                getLog().debug( "Selected version:" + selectedVersion.toString() );
+                ArtifactVersion selectedVersion = new DefaultArtifactVersion(version);
```
Comment (olamy) [inline src/main/java/org/codehaus/mojo/versions/UseLatestReleasesMojo.java]: why changing code format?

Comment (obfischer) [reply src/main/java/org/codehaus/mojo/versions/UseLatestReleasesMojo.java]: I will keep the original formatting.

**Comment by obfischer** [comment]
Hi @olamy, could you please have look at my PR again?

**Comment by khmarbaise** [comment]
@olamy What do you think now. From my point of view it's an improvement...?

**Comment by olamy** [comment]
@obfischer sorry for delay. LGTM so merging it.
Thanks a lot!

**Pull request merged**